### ISSUE TITLE
Uncaught error in transactionCollector#collect()

### ISSUE THREAD

**Issue by katat** [nervosnetwork/lumos#2]
The `cellCollector` works great and fast. Awesome works!

 I am trying to test the `TransactionCollector` to get all the transactions with a lock script. However, it throws an error as below:
```
(node:81064) UnhandledPromiseRejectionWarning: ReferenceError: RPC is not defined
    at new TransactionCollector (/Users/kata/repos/dev/nervos/test-lumos/node_modules/@ckb-lumos/indexer/lib/index.js:204:20)
    at /Users/kata/repos/dev/nervos/test-lumos/index.js:35:34
(node:81064) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:81064) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I guess there is a compatibility issue in the lumos calls to the `ckb-indexer`.

Here is the test code:
https://github.com/katat/lumos-performance-test/blob/2afa9f043e37865f97c3e042933076b1d707aae1/index.js#L41

**Comment by xxuejie** [comment]
This should now be fixed in version 0.1.7 of `@ckb-lumos/indexer`

**Comment by katat** [comment]
Thanks for the quick update.

With the latest version, it now does not throw the error. 
But it never returns the results for the transactions with lock args `0xdde7801c073dfb3464c7b1f05b806bb2bbb84e99`, even after 2 mins of executing.

I believe this version already fixes this RPC compatibility issue. So I am closing this one and creating another one for the query hanging issue.
